pkgs/applications/misc/udiskie/default.nix

```diff
@@ -0,0 +1,47 @@
+{ stdenv, fetchFromGitHub, asciidoc-full, gettext
+, gobjectIntrospection, gtk3, hicolor_icon_theme, libnotify
+, pythonPackages, udisks2, wrapGAppsHook }:
+
+pythonPackages.buildPythonApplication rec {
+  name = "udiskie-${version}";
+  version = "1.5.1";
+
+  src = fetchFromGitHub {
+    owner = "coldfix";
+    repo = "udiskie";
+    rev = version;
+    sha256 = "01x5fvllb262x6r3547l23z7p6hr7ddz034bkhmj2cqmf83sxwxd";
+  };
+
+  preConfigure = ''
+    export XDG_RUNTIME_DIR=/tmp
```

Comment:
Yes I agree that would be much nicer. We should build wrappers in such a way as well I think.

But for the case here, since at this point the state is inherited from one phase to another, there is no difference between using `export X=y;` or `X=y;` except if you want to export the var after a certain phase.